src/Pyrexx/ZUGFeRD/Model/Trade/Settlement.php

```diff
@@ -45,12 +46,19 @@ class Settlement
     private $tradeTaxes = array();
 
     /**
-     * @var MonetarySummation
-     * @Type("Pyrexx\ZUGFeRD\Model\Trade\MonetarySummation")
-     * @XmlElement(cdata = false, namespace = "urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:12")
-     * @SerializedName("SpecifiedTradeSettlementMonetarySummation")
+     * @var AllowanceCharge[]
+     * @Type("array<Pyrexx\ZUGFeRD\Model\AllowanceCharge>")
+     * @XmlList(inline=true, entry="SpecifiedTradeAllowanceCharge", namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:12")
      */
-    private $monetarySummation;
+    private $allowanceCharges = [];
+
+    /**
+     * @var SpecifiedLogisticsServiceCharge
+     * @Type("Pyrexx\ZUGFeRD\Model\Trade\SpecifiedLogisticsServiceCharge")
+     * @XmlElement(cdata=false, namespace="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:12")
+     * @SerializedName("SpecifiedLogisticsServiceCharge")
+     */
+    private $logisticsService;
```

Comment:
may be name the property logisticsServiceCharge for consitency? and getter and setter as well

Reply:
Hi, sorry for the delay, just renamed **logisticsService** property.